### PR TITLE
Added integration tests on webb.js for mixer and anchor

### PR DESCRIPTION
**Pull request by AhmedKorim**

### CODE REVIEW

**Comment by drewstone** [comment]
Can you please fix the spelling of `proving-manger` file to `proving-manager` @AhmedKorim. I'd like to get the spelling fixed in this PR.

File: tests/integration_tests/substrate/anchorTests.ts [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+}
+
+describe.only('Anchor tests', function () {
+  // increase the timeout for relayer tests
+  this.timeout(60 * 60 * 1000);
+
+  before(async function () {
+    // If LOCAL_NODE is set the tests will continue  to use the already running node
+    if (true) {
+      nodes = startDarkWebbNode();
```
Comment (drewstone) [inline tests/integration_tests/substrate/anchorTests.ts:53]: Should fix this to `startWebbNode()`

File: tests/package.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
+{
+  "name": "webb.js-tests",
```
[the remark this answers is not in the document]
Comment (AhmedKorim) [reply tests/package.json]: The is like "${package-name}-tests", but what you mentioned sounds good

File: examples/sdk-mixer/package.json [hunk 3]
```diff
@@ -8,12 +8,13 @@
...
   },
   "dependencies": {
-    "@polkadot/api": "^6.9.2",
-    "@webb-tools/api": "^0.1.4-7",
-    "@webb-tools/app-util": "^0.1.4-7",
-    "@webb-tools/sdk-core": "^0.1.4-7",
-    "@webb-tools/sdk-mixer": "^0.1.4-7",
-    "@webb-tools/types": "^0.1.4-7"
+    "@polkadot/api": "^7.5.1",
+    "@webb-tools/type-definitions": "file:../packages/type-definitions/build",
```
[the remark this answers is not in the document]
Comment (AhmedKorim) [reply examples/sdk-mixer/package.json:12]: Will handle tests in a better way on a next PR

**Pull request merged**